vsintegration/src/FSharp.Editor/CodeLens/LineLens.fs

```diff
@@ -350,10 +350,29 @@ type internal FSharpCodeLensService
             let! taggedText, navigation = lens.TaggedText
             let textBox = new TextBlock(Width = 500., Background = Brushes.Transparent, Opacity = 0.0, TextTrimming = TextTrimming.None)
             DependencyObjectExtensions.SetDefaultTextProperties(textBox, formatMap.Value)
-            textBox.Inlines.Add (Documents.Run Settings.CodeLens.Prefix)
+
+            let prefix = Documents.Run Settings.CodeLens.Prefix
+            prefix.Foreground <- SolidColorBrush(Color.FromRgb(153uy, 153uy, 153uy))
+            textBox.Inlines.Add (prefix)
+
             for text in taggedText do
+
+                let properties = layoutTagToFormatting text.Tag
+                let properties =
```

Comment:
merge those two definitions, it's not needed to this here.

Reply:
How would you want them merged? The first definition of `properties` is used several places in the second definition. Would a name change to the first one suffice?

Reply:
Fixed :)